File: 1476.c
// Calculate to Date

#include <stdio.h>

int main(){
    int e,s,m;
    int n=1,a=1,b=1,c=1;
    scanf("%d %d %d", &e, &s, &m);

    while(1){
        if(a!=e || b!=s || c!=m){
            n++;a++;b++;c++;
            if(a>15) a=1;
            if(b>28) b=1;
            if(c>19) c=1;
        }
        else break;
    }
    printf("%d",n);
    return 0;
}

File: 1476.py
# Calculate to Date

e,s,m=map(int,input().split())
n=1;a=1;b=1;c=1
while 1:
 if a!=e or b!=s or c!=m:
  n+=1;a+=1;b+=1;c+=1
  if a>15:a=1
  if b>28:b=1
  if c>19:c=1
 else:break
print(n)